pkgs/development/rocm-modules/6/rocm-bandwidth-test/default.nix

```
@@ -0,0 +1,38 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  rocmPackages,
+  cmake,
+}:
+
+stdenv.mkDerivation (finalAttrs: rec {
```

```suggestion
stdenv.mkDerivation (finalAttrs: {
```

Using both `finalAttrs` and `rec` is essentially the same thing twice over

---

fixed